tests/test_degrade.py

```diff
@@ -111,60 +112,122 @@ def test_degrade_map_recarray(self):
 
     def test_degrade_widemask_or(self):
         """
-        Test HealSparse.degrade functionality with WIDE_MASK
+        Test HealSparse.degrade OR functionality with WIDE_MASK
         """
-        
+
         nside_coverage = 32
         nside_map = 256
         nside_map2 = 64
         sparse_map = healsparse.HealSparseMap.make_empty(nside_coverage, nside_map,
-                                                 WIDE_MASK, wide_mask_maxbits=7)
+                                                         WIDE_MASK, wide_mask_maxbits=7)
         sparse_map_or = healsparse.HealSparseMap.make_empty(nside_coverage, nside_map2,
-                                                 WIDE_MASK, wide_mask_maxbits=7)
+                                                            WIDE_MASK, wide_mask_maxbits=7)
         # Fill some pixels in the "high-resolution" map
         pixel = np.arange(4000, 8000)
         sparse_map.set_bits_pix(pixel, [4])
-        
+
         # Check which pixels will be full in the "low-resolution" map and fill them
-        pixel2 = np.unique(np.right_shift(pixel, healsparse.utils._compute_bitshift(nside_map2, nside_map))) 
+        pixel2 = np.unique(np.right_shift(pixel, healsparse.utils._compute_bitshift(nside_map2, nside_map)))
         sparse_map_or.set_bits_pix(pixel2, [4])
-        
+
         # Degrade with or
         sparse_map_test = sparse_map.degrade(nside_map2, reduction='or')
-        
+
         # Check the results
         testing.assert_almost_equal(sparse_map_or._sparse_map, sparse_map_test._sparse_map)
-        
+
     def test_degrade_widemask_and(self):
         """
-        Test HealSparse.degrade functionality with WIDE_MASK
+        Test HealSparse.degrade AND functionality with WIDE_MASK
         """
-        
+
         nside_coverage = 32
         nside_map = 256
         nside_map2 = 64
         sparse_map = healsparse.HealSparseMap.make_empty(nside_coverage, nside_map,
-                                                 WIDE_MASK, wide_mask_maxbits=7)
+                                                         WIDE_MASK, wide_mask_maxbits=7)
```

Comment:
The and/or wide_mask tests need two separate tests each, unfortunately, one of them with < 8 bits (like this) and another with > 8 bits.  The one with > 8 bits needs to ensure that some pixels have a low bit set, some pixels have a high bit set, and some have both.  (I have learned this the hard way!)

Reply:
Great suggestion, this allowed me to catch a couple of bugs.